fix: Add Prisma docs on how to connect to postgres (#20503)
supabase/supabase: apps/docs/pages/guides/database/connecting-to-postgres.mdx

  </StepHikeCompact.Step>

</StepHikeCompact>
### Connecting with Prisma

> [Prisma](https://www.prisma.io/) is a TypeScript ORM for SQL databases focused on .

Reviewer: ```suggestion
[Prisma](https://www.prisma.io/) is another popular TypeScript ORM for SQL databases.
```

> Currently there's a limitation where you can't use prepared statements with Prisma and Supabase. This is a known issue and we're working on the fix.

Reviewer: ```suggestion

<Admonition type="note">

Currently there's a limitation where you can't use prepared statements with Prisma and Supabase. This is a known issue and we're working on the fix.

</Admonition>

```

<StepHikeCompact>
  <StepHikeCompact.Step step={1}>

    <StepHikeCompact.Details title="Set your connection pool to Transaction">

>     Change the connection pool in your project to `Transaction` by following changing it in the [Pool Mode](https://supabase.com/dashboard/project/_/settings/database) dropbox.

Reviewer: ```suggestion
    Make sure your connection pool mode is set to `Transaction` in your [Database Settings](https://supabase.com/dashboard/project/_/settings/database).
```

    </StepHikeCompact.Details>
    <StepHikeCompact.Code>

>      ![Change Connection pool to Transaction](/docs/img/guides/database/pool-mode-transaction.png)

Reviewer: ```suggestion
     ![Change Connection pool to Transaction](/docs/img/database/pool-mode-transaction.png)
```
